Firmata.cpp

```diff
@@ -398,7 +437,11 @@ void FirmataClass::attach(uint8_t command, systemCallbackFunction newFunction)
  */
 void FirmataClass::attach(uint8_t command, stringCallbackFunction newFunction)
 {
-    parser.attach(command, (stringCallbackFunction)newFunction);
+  switch (command) {
```

Comment:
Does this need to be a switch statement? Would an if statement be more performant (I'm not sure if an if or switch statement is better performance wise when there is only a single condition)? I don't foresee having other types of string callback functions.